io.github.tlcfem.suanPan.yml

```diff
@@ -0,0 +1,60 @@
+app-id: io.github.tlcfem.suanPan
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: suanPan
+finish-args:
+  - '--filesystem=home'
+  - '--socket=fallback-x11'
+  - '--device=dri'
+  - '--socket=wayland'
+  - '--share=ipc'
+cleanup:
+  - /bin/vtk*
+  - /include
+  - /lib/cmake
+  - /lib/libvtk*
+  - /lib/vtk*
+  - /share/vtk*
+  - /share/licenses
+modules:
+  - name: suanPan
+    buildsystem: cmake
+    builddir: true
+    config-opts:
+      - '-DCMAKE_BUILD_TYPE=Release'
+      - '-DBUILD_MULTITHREAD=ON'
+      - '-DUSE_HDF5=ON'
+      - '-DUSE_EXTERNAL_VTK=ON'
+    post-install:
+      - install -Dm644 io.github.tlcfem.suanPan.metainfo.xml /app/share/metainfo/io.github.tlcfem.suanPan.metainfo.xml
+      - install -Dm644 suanPan-ua.svg /app/share/icons/hicolor/128x128/apps/io.github.tlcfem.suanPan.svg
+      - install -Dm644 io.github.tlcfem.suanPan.desktop /app/share/applications/io.github.tlcfem.suanPan.desktop
+    sources:
+      - type: git
+        url: 'https://github.com/TLCFEM/suanPan.git'
+        tag: 'suanPan-v2.6.1'
+      - type: file
+        url: >-
+          https://raw.githubusercontent.com/TLCFEM/suanPan/dev/Resource/io.github.tlcfem.suanPan.metainfo.xml
+        sha256: 60d949178b8fb208495fc21e3609265dd197d657f748cf29ef726e3b6f7c0c00
+      - type: file
+        url: >-
+          https://raw.githubusercontent.com/TLCFEM/suanPan/dev/Resource/suanPan-ua.svg
```

Comment:
why do you need this? Isn't it in the tree checkout above?

also this reference a branch instead of a commit.

but since it seems to be specifically for this, why not naming the file properly in the first place?